### PR TITLE
Fix sensitivity calculation

### PR DESCRIPTION
**Pull request by geofjamg**
Signed-off-by: Geoffroy Jamgotchian <geoffroy.jamgotchian@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Bug fix

**What is the current behavior?** *(You can also link to an open issue here)*
Even if there is no error in sensi calculation, calculation is not properly done. 

**What is the new behavior (if this is a feature change)?**
Sensi calculation is better implemented.

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

**Review by Hadrien-Godard — APPROVED** [review]

File: src/main/java/com/powsybl/openloadflow/ac/equations/ClosedBranchSide1ReactiveFlowEquationTerm.java [hunk 1]
```diff
@@ -34,10 +34,13 @@ protected double calculateSensi(double dph1, double dph2, double dv1, double dv2
         double theta = theta1(ksi, ph1(), a1(), ph2());
         double cosTheta = FastMath.cos(theta);
         double sinTheta = FastMath.sin(theta);
+        double cosKsi = FastMath.cos(ksi);
         return dq1dph1(y, v1, r1, v2, sinTheta) * dph1
                 + dq1dph2(y, v1, r1, v2, sinTheta) * dph2
-                + dq1dv1(y, FastMath.cos(ksi), b1, v1, r1, v2, cosTheta) * dv1
-                + dq1dv2(y, v1, r1, cosTheta) * dv2;
+                + dq1dv1(y, cosKsi, b1, v1, r1, v2, cosTheta) * dv1
```
Comment (AnkurArohi) [inline src/main/java/com/powsybl/openloadflow/ac/equations/ClosedBranchSide1ReactiveFlowEquationTerm.java:40]: @geofjamg 
Is this not reactive power sensitivity calculation, based on the nodal voltage variations, at this particular node.
One would just couple it up with the admittance matrix to get all nodes based reactive power sensitivity,  your opinion?

Comment (geofjamg) [reply src/main/java/com/powsybl/openloadflow/ac/equations/ClosedBranchSide1ReactiveFlowEquationTerm.java:40]: This is part of reactive power sensitivity calculation. 
This is what is done bit it is managed at the `Equation` level.

Comment (AnkurArohi) [reply src/main/java/com/powsybl/openloadflow/ac/equations/ClosedBranchSide1ReactiveFlowEquationTerm.java:40]: This is exactly what would be intresting,just the management at equation level should be coupled with the admittance matrix in order to be able to determine the reactive power sensitivity at he NVKP level.
This is similar to GLSK factor for the reactive power.

Comment (AnkurArohi) [reply src/main/java/com/powsybl/openloadflow/ac/equations/ClosedBranchSide1ReactiveFlowEquationTerm.java:40]: Can this info be ported out?

**Pull request merged**